zfp.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: zfp
+  version: 1.0.1
+  epoch: 0
+  description: zfp is a compressed format for representing multidimensional floating-point and integer arrays.
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/LLNL/zfp
+      tag: ${{package.version}}
+      expected-commit: f40868a6a1c190c802e7d8b5987064f044bf7812
+
+  - uses: cmake/configure
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: LLNL/zfp
```

Comment:
I'd like to have a test here showing the library/binary in use. I think the cmake command creates a test script - perhaps you can add it in the test pipeline and check the results? There are a decent number of other packages that use the test pipeline to copy examples from

Reply:
Added this